src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1526,6 +1527,28 @@ public String toVariableNameString(EnchantmentOffer eo) {
 				.name("Transform Reason")
 				.description("Represents a transform reason of an <a href='events.html#entity transform'>entity transform event</a>.")
 				.since("2.8.0"));
+
+		Classes.registerClass(new ClassInfo<>(WorldBorder.class, "worldborder")
+			.user("world ?borders?")
+			.name("World Border")
+			.description("Represents the border of a world.")
+			.since("INSERT VERSION")
+			.parser(new Parser<WorldBorder>() {
+				@Override
+				public boolean canParse(ParseContext context) {
+					return false;
+				}
+
+				@Override
+				public String toString(WorldBorder border, int flags) {
+					return "world border of " + (border.getWorld() != null ? " of world named " + border.getWorld().getName() : "");
+				}
+
+				@Override
+				public String toVariableNameString(WorldBorder border) {
+					return toString(border, 0);
```

Comment:
might want to have a more compressed and guaranteed unique format for variables:
```[virtual] worldborder of player/world``` for example.

Reply:
I'm not really sure if I can get a player's name/uuid here but I can add the virtual part